### PR TITLE
Added simple readme

### PR DESCRIPTION
**Pull request by damcclean**
#19

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Webslides
+
+> Hosting Reveal.js markdown-based presentations as a Service
+
+## Install
+
+To install Webslides, you'll first need a few pre-requisites:
+
+* Web server - Ngnix or Apache will do just fine
```
Comment (m1guelpf) [inline readme.md]: Do you really? You can use artisan's server here or do strange things... I'm tempted to just copy OrgManager's setup guide here and forget about writing a comprehensive one here

Comment (damcclean) [reply readme.md]: That'd be easier :) lol

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* PHP - PHP5 will work, but PHP7 is highly recommended
```
Comment (m1guelpf) [inline readme.md]: Doesn't Laravel 5 require PHP7? Anyway, PHP5 is being deprecated soon, so PHP7 lol

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Composer
+* NPM/Yarn - depends on personal prefrence
```
Comment (m1guelpf) [inline readme.md]: lol

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* MySQL database
```
Comment (m1guelpf) [inline readme.md]: Not really, you can use postgres or even sqlite

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+* Git
```
Comment (m1guelpf) [inline readme.md]: Or you could download a zip :)

**Pull request closed without merging**